arch/xtensa/src/esp32/Make.defs

```diff
@@ -111,8 +111,8 @@ CHIP_CSRCS += esp32_himem.c
 CHIP_CSRCS += esp32_himem_chardev.c
 endif
 
-ifeq ($(CONFIG_ESP32_EFUSE),y)
 CHIP_CSRCS += esp32_efuse.c
```

Comment:
Why compile ESP32_EFUSE when CONFIG_ESP32_EFUSE is not enabled ?

Reply:
Because the chip revision is read from efuses, but we don't need to implement all efuses features, so I modified `esp32_efuse.c` and added the `CONFIG_ESP32_EFUSE` to build the required sources when all efuse features are required.

Reply:
Ok